src/GlobalHeader/index.js

```diff
@@ -134,280 +142,267 @@ const GlobalHeader = ({ globalNav, pages, docs, location, ...props }) => {
-                            'spectrum-Popover--bottom',
-                            'spectrum-Dropdown-popover',
-                            'spectrum-Dropdown-popover--quiet',
-                            {
-                              'is-open': isPrimary
-                                ? openPrimaryMenu
-                                : openSecondaryMenu
-                            }
-                          )}
-                          css={css`
-                            display: block;
-                            padding: var(
-                              --spectrum-global-dimension-static-size-300
-                            );
-                            z-index: 2;
-                            max-width: none !important;
-                          `}
-                        >
-                          <div
-                            css={css`
-                              display: flex;
-                              gap: var(
-                                --spectrum-global-dimension-static-size-200
-                              );
-                            `}
-                          >
-                            {menu.sections.map((section, i) => (
-                              <React.Fragment key={i}>
-                                <View>
-                                  {section.heading && (
-                                    <View
-                                      marginBottom='size-200'
-                                      marginStart='size-200'
-                                    >
-                                      <strong className='spectrum-Heading--XS'>
-                                        {section.heading}
-                                      </strong>
-                                    </View>
-                                  )}
-                                  <ul
-                                    className='spectrum-AssetList'
-                                    role='menu'
+                        }}
+                      >
+                        {menu.title}
+                      </PickerButton>
+                      <Popover
+                        ref={isPrimary ? primaryPopover : secondaryPopover}
+                        id={id}
+                        variant='picker'
+                        isQuiet
+                        isOpen={isPrimary ? openPrimaryMenu : openSecondaryMenu}
+                        css={css`
+                          display: block;
+                          padding: var(
+                            --spectrum-global-dimension-static-size-300
+                          );
+                          z-index: 2;
+                          max-width: none !important;
+                          max-height: none !important;
+                          width: auto !important;
+                        `}
+                      >
+                        <Flex>
+                          {menu.sections.map((section, i) => (
```

Comment:
This should be wrapped in a `nav` component to pass a11y tests.

` <nav aria-label="Secondary">`